json/src/main/kotlin/com/trib3/json/ObjectMapperProvider.kt

```diff
@@ -41,6 +52,20 @@ class ObjectMapperProvider @Inject constructor(
         mapper.registerModule(KotlinModule())
         mapper.registerModule(MetricsModule(TimeUnit.SECONDS, TimeUnit.SECONDS, false))
         mapper.registerModule(ThreeTenExtraModule())
+
+        // set up guice <-> jackson inject bridge like jackson's guice module does
```

Comment:
might want to bump version for this in case any unintended side effects?

Reply:
This is a bit over my head but the risk of "harmful" side effects seems low/nil I think your right it'll probably be fine without the bump.